xarray/core/formatting.py

```diff
@@ -253,8 +253,15 @@ def summarize_coord(name, var, col_width):
 
 
 def summarize_attr(key, value, col_width=None):
-    # ignore col_width for now to more clearly distinguish attributes
-    return u'    %s: %s' % (key, maybe_truncate(value))
+    """Summary for __repr__ - use ``X.attrs[key]`` for full value."""
+    key = u'    %s:' % unicode_type(key)
+    if col_width is not None:
+        key = pretty_print(key, col_width)
+    output = u'%s %s' % (key, unicode_type(value).replace(u'\t', u'    '))
```

Comment:
I disagree - it's just adjusting the spacing and replacing tabs and newlines before printing them.  While I don't think numbers or array representations would have spaces (and arrays with newlines are rare), surely we want to process them for display in the same way?